app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.ts

```diff
@@ -614,18 +614,26 @@ function _filterPipelineList(
  * @param existingSchedules - the existing enabled trigger schedules.
  */
 function _transformSchedule(existingSchedules: ISchedule[]) {
+  // Only get the PROGRAM_STATUS, AND, OR types schedules
+  const filteredTriggerSchedules = existingSchedules.filter(
+    (schedule) =>
+      schedule.trigger.type === PipelineTriggersTypes.programStatus ||
```

Comment:
aren't these all the pipeline triggers types? https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/components/PipelineTriggers/store/PipelineTriggersTypes.ts

Reply:
if those are all the triggers types remove this filter?

Reply:
ahh nvm i see whats going on here